.rubocop.yml

```
@@ -15,7 +15,7 @@ Govuk:
   Exclude:
     - "app/components/trainees/confirmation/degrees/view.html.erb"
     - "app/components/personas/view.html.erb"
-
+    - "app/components/button_link_to/view.rb"
```

```suggestion
Govuk/GovukLinkTo:
  Exclude:
    - "app/components/button_link_to/view.rb"

```

Needs to be more selective, otherwise it become an oversight.

---

thanks, fixed